ceph_salt/config_shell.py

```diff
@@ -315,6 +315,12 @@ def possible_values(self):
                     },
                 }
             },
+            'Registries': {
+                'type': 'list',
+                'default': [],
+                'help': "List of custom registries",
```

Comment:
maybe `List of custom registries in V2 format` ?

Reply:
Is it possible to also validate that the list elements need to be `dict`?

Reply:
> maybe `List of custom registries in V2 format` ?

Fixed.

> Is it possible to also validate that the list elements need to be `dict`?

Hm..it's actually a JSON string. Let's keep it as it is refine it later, ideally we should change the CLI to the form
```
ceph-salt config /Containers/Registries add prefix=registry.opensuse.org location=192.168.2.106:5000/registry.opensuse.org insecure=true
```